sale_comment_propagation/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================================
+Comments for sale documents (order, picking and invoice)
+========================================================
+
+With this module you can add specific comments to a customer, for sale order,
+delivery order and invoices. Part of the info will be passed from one to other.
+Those data will be automatically added to each item.
+
```

Comment:
Please add a comment that this module don't add this comments to any report. A user could expect that.